### ISSUE TITLE
Options for VPC Use

### ISSUE THREAD

**Issue by craigbruce** [jtriley/StarCluster#366]
Using 0.95 I generated a new config and edited my cluster stanza to include the required options:

SUBNET_ID=subnet-XXXXXX
PUBLIC_IPS=False

However, when the ec2 instances boot they **do** get public IPs and the step where starcluster attempts to SSH in fails.

If I use the same file but launch with: `starcluster start -N subnet-XXXXXXX --no-public-ips mycluster`. It works as expected, no public IPs, ssh access is gained the cluster is configured.

Cheers
Craig

**Comment by jtriley** [comment]
@craigbruce Fixing this now. Thanks for reporting!

**Comment by craigbruce** [comment]
I've just tested this but still have the same results:

```
>>> Using default cluster template: smallcluster
>>> Validating cluster template settings...
>>> Cluster template settings are valid
>>> Starting cluster...
>>> Launching a 2-node cluster...
>>> Creating security group @sc-vpcdev...
Reservation:r-3bca9d1b
```

Using the -N options:

```
>>> Using default cluster template: smallcluster
>>> Validating cluster template settings...
>>> Cluster template settings are valid
>>> Starting cluster...
>>> Launching a 2-node cluster...
>>> Creating security group @sc-vpcdev...
*** WARNING - PUBLIC IPS HAVE BEEN DISABLED!!!

This means StarCluster must be executed from a machine within the cluster's VPC
(vpc-XXXX) otherwise it will hang forever trying to connect to the instances.
Reservation:r-06f6a126
```

**Comment by jtriley** [comment]
@craigbruce I think I just found the last remaining culprit. When adding the config setting to static.py I incorrectly set the type to 'str' instead of 'bool'. I'm fixing this now along with another bug and releasing another patch release `0.95.2` today.

**Comment by craigbruce** [comment]
I can confirm this is fixed in 0.95.2 - thanks!

**Comment by jtriley** [comment]
@craigbruce Awesome, thanks for confirming!
